addons/web/static/tests/views/list_view_tests.js

```diff
@@ -11244,9 +11239,11 @@ QUnit.module("Views", (hooks) => {
             "There should be an element creating the indentation for the subgroup."
         );
         assert.notStrictEqual(
-            target.querySelector("tr:nth-child(1) th.o_group_name .fa").style.paddingLeft,
+            getComputedStyle(target.querySelector("tr:nth-child(1) th.o_group_name .fa"))
+                .getPropertyValue("--o-list-group-level")
+                .trim(),
```

Comment:
Let's reuse the function `getCssVar()` you have introduced above.

Reply:
I extracted the getCssVar and made it more generic (no more internal querying of "span" element).